config/config.go

```diff
@@ -44,6 +44,7 @@ type Config struct {
 	OTExporterOTLPEndpoint     string        `envconfig:"OTEL_EXPORTER_OTLP_ENDPOINT"`
 	OTServiceName              string        `envconfig:"OTEL_SERVICE_NAME"`
 	OTBatchTimeout             time.Duration `envconfig:"OTEL_BATCH_TIMEOUT"`
+	OtelEnabled                bool          `envconfig:"OTEL_ENABLED"`
```

Comment:
Shouldn't there be a corresponding `config_test.go` change too?

Reply:
I've added this in to the tests now.